src/app/components/student-list/student-list.component.ts

```diff
@@ -14,43 +14,51 @@ import type { StudentRecordDisplayComponent } from '../student-record-display/st
     styleUrls: ['./student-list.component.sass']
 })
 export class StudentListComponent implements CourseConfigurable {
-    //drop down
+    // Possible values for the number of students displayed per page in the student list
     public POSSIBLE_PAGE_CNTS = [5, 10, 25, 50];
+    //defult number of students displayed is 50
     public DEFAULT_PAGE_CNT = 50;
-
+    //current select course and can be none
     course?: Course;
+    // Configuration related to token ATM
     configuration?: TokenATMConfiguration;
+    //Student data in the formate of paginatedView which store student data is previous current and next page format
     students?: PaginatedView<Student>;
+    // Link student names with their grades
```

Comment:
This Map links student id to their grades since name is not a unique identifier.

Reply:
Resolved in the new commit.